prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/components/Partial.jsx

```diff
@@ -33,10 +33,15 @@ const Partial = ({ params: { moduleName }, location: { query }, postProps }) =>
 
 const propSelector = (state) => state.getIn(['modules', 'frank-lloyd-root', 'postProps'], iMap()).toJS();
 
-export const onPartialRouteEnter = ({ dispatch, getState }) => (nextState, replace, cb) => {
+export const onPartialRouteEnter = (
+  { dispatch, getState },
+  partialOnly = false,
+  textOnly = false
```

Comment:
I have reused a lot of the existing functionality from the `renderPartial` mechanism to implement the render module as a text implementation here to avoid duplicated code, however it might not be clear how to implement it individually, please let me know what you think about leaving it as it is or moving it to its own component

Reply:
as long as we have it documented this is okay